VultisigApp/VultisigApp/Services/RpcServiceActor.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  RpcServiceActor.swift
+//  VultisigApp
+//
+//  Refactored to use Actor for thread-safety
+//
+
+import Foundation
+import BigInt
+
+actor RpcServiceActor {
```

Comment:
RpcService doesn't have any state , so it should not be an actor

Only use actor when it has a shared state that need to threadsafe